src/legacy/ui/public/saved_objects/helpers/apply_es_resp.ts

```diff
@@ -0,0 +1,83 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import _ from 'lodash';
+import { EsResponse, SavedObject, SavedObjectConfig } from 'ui/saved_objects/types';
+import { parseSearchSource } from 'ui/saved_objects/helpers/parse_search_source';
+import { expandShorthand, SavedObjectNotFound } from '../../../../../plugins/kibana_utils/public';
+import { IndexPattern } from '../../../../core_plugins/data/public';
+
+/**
+ * A given response of and ElasticSearch containing a plain saved object is applied to the given
+ * savedObject
+ */
+export async function applyESResp(
+  resp: EsResponse,
+  savedObject: SavedObject,
+  config: SavedObjectConfig
+) {
+  const mapping = expandShorthand(config.mapping);
+  const esType = config.type || '';
+  savedObject._source = _.cloneDeep(resp._source);
+  const injectReferences = config.injectReferences;
+  const hydrateIndexPattern = savedObject.hydrateIndexPattern!;
+
+  if (resp.found != null && !resp.found) {
```

Comment:
what exactly are you trying to check here? if  `resp.found` has boolean value of false?

Reply:
I've investigated this, I've copied the code, turns up this isn't even necessary, so I will clean this up, thank you!